src/main/java/ch/wisv/areafiftylan/service/repository/EventRepository.java

```diff
@@ -0,0 +1,17 @@
+package ch.wisv.areafiftylan.service.repository;
+
+import ch.wisv.areafiftylan.model.Event;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+/**
+ * Created by Sille Kamoen on 27-3-16.
+ */
+public interface EventRepository extends JpaRepository<Event, Long> {
+
+    Optional<Event> findById(Long eventId);
```

Comment:
The findOne method already does this. That doesn't use a wrapper though, but isn't checking a null return a good substitute?